GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -108,7 +108,13 @@ public partial class FormBrowse : GitModuleForm, IBrowseRepo
 
         private readonly TranslationString _commitButtonText =
             new TranslationString("Commit");
-        #endregion
+
+        private readonly TranslationString _undoLastCommitText =
+            new TranslationString("All the changes of the commit will be kept in the staging area.\n\nAre you sure you want to undo the last commit?");
```

Comment:
How about the following?

_You can find all of the commit's changes in the staging area._
_Proceed?_

Reply:
done.